.faktory/conf.d/cron.toml

```diff
@@ -9,7 +9,7 @@
     type = "TranslateAuditCronJob"
     queue = "critical"
 [[cron]]
-  schedule = "*/5 * * * *"
+  schedule = "* * * * *"
```

Comment:
As noted in the PR, this is an effectively unrelated change, but I feel like having this job run every minute instead of every 5 minutes just makes things a lot easier to test!